elementor/elementor
# Fix: Set padding on icon span instead of text span (#9831) [ED-9757]

## PR Checklist



## PR Type
What kind of change does this PR introduce?


## Summary

This PR can be summarized in the following changelog entry:

* Text-Indent in Icon List widget items applies padding to icon element instead of text element

## Description
An explanation of what is done in this PR

* When a list item doesn't have an icon, the anchor (if the list item has one) will have blank space padded on the left because the text-indent control would add padding to the left of the `<span>` that contains the text. This fixes that because the icon element does not output unless there is an icon selected, which means there will be no padding applied to the anchor tag in the event of a list item not having an icon selected.

## Test instructions
This PR can be tested by following these steps:

* Creating an Icon List widget and setting the layout to `inline`
* Remove an icon from any list item
* Add a number to the _Text Indent_ control
* Preview the page
* Verify that the number added to the _Text Indent_ control is added as CSS to the `.elementor-icon-list-icon` class as `padding-right` (in LTR languages, left in RTL languages) instead of the previous implementation of adding `padding-left` to the `.elementor-icon-list-text` element.

## Quality assurance


Fixes #9831
